sxcu/sxcu.py

```diff
@@ -143,11 +143,11 @@ def upload_file(
         url = join_url(self._get_api_endpoint(default_domain=False), "/files/create")
         if isinstance(file, io.BytesIO):
             files = {"file": file}
-            res = request_handler.post(url, files=files, data=data)
         else:
-            with open(file, "rb") as img_file:
-                files = {"file": img_file}
-                res = request_handler.post(url, files=files, data=data)
+            files = {"file": open(file, "rb")}
```

Comment:
maybe wrap this in a try/except?

Reply:
oh sorry, I never got time to fix that. I'll try to fix these and merge them by tomorrow.

Reply:
sounds good

Reply:
I ended up forgetting this again, I'll work on this next week I guess.